srcpkgs/asahi-base/files/README.voidlinux

```diff
@@ -0,0 +1,31 @@
+For some time, the kernel, mesa and uboot packages will have Asahi
+as upstream.  The Asahi Linux documentation is an excellent source.
+Here we discuss Void Linux specific aspects only.
+
+Use Asahi Linux install scripts with "UEFI environment only", then proceed
+with usual Void installation.  The default is to have EFI partition at
+/boot, other choices can be made by editing m1n1 config.
```

Comment:
hum i see

Is there a reason to do it this way?

Reply:
no particular one, just the one i'm used to.

Reply:
but as above it is configurable.

Reply:
i think this needs to be discussed further. While it is technically fine, it is also nonstandard.

All the void documentation, the void installer mount, and the asahi installer mount ESP to /boot/efi

just became its the way you're used, to isn't a good reason

Reply:
void's preferred/supported config is for ESP at /boot/efi.

Reply:
Ok, will do that.